src/visualizers/Chaos.ts

```diff
@@ -127,6 +147,11 @@ const paramDesc = {
             'Alpha factor (from 0.0=transparent to 1.0=solid) of the dots.',
         visibleDependency: 'dummyDotControl',
         visibleValue: true,
+        validate(a: number, status: ValidationStatus) {
+            if (a < 0 || a > 1) {
+                status.addError('must be between 0 and 1, inclusive.')
```

Comment:
above you didn't use a period

Reply:
OK, I tried to make all of the categories of text related to parameters and validations each consistent:
* parameter description fields should now all be complete sentences, with capital letters and periods.
* individual-field validation messages should now all be uncapitalized phrases without periods.
* object-wide messages that appear at the top of the parameter tab rather than by a specific field should now all be complete sentence, w/ caps and periods.

Do those seem like reasonable conventions? Should we put those conventions in the documentation somewhere? If so, do you have a suggestion for where?

Reply:
Those seems reasonable to me.

Reply:
For documentaion, maybe in the example visualizer code?